wiki/Welcome/vi.md

```diff
@@ -0,0 +1,47 @@
+# Chào mừng
+
+Chào mừng đên osu!, một trò chơi nhịp điệu được lập trình bởi peppy với 4 chế độ chơi: osu!, chế độ bấm vòng tròn; osu!taiko, chế độ đánh trống; osu!catch, chế độ hứng quả; và osu!mania, chế độ bấm phím.
+
+## Một vài điều cơ bản
+
+- [Cài đặt](/wiki/Installation)
+- [Đăng kí](/wiki/Registration)
+- [Luật lệ](/wiki/Rules)
+- [FAQ](/wiki/FAQ)
+- [Chế độ chơi](/wiki/Game_mode)
+  - ![osu! mode icon](/wiki/shared/mode/osu.png) [osu!](/wiki/Game_mode/osu!)
+  - ![osu!taiko icon](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Game_mode/osu!taiko)
+  - ![osu!catch icon](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Game_mode/osu!catch)
+  - ![osu!mania icon](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Game_mode/osu!mania)
+- [Chế độ nhiều người chơi](/wiki/Multi)
+- [Tiêu chuẩn xếp hạng](/wiki/Ranking_Criteria)
+  - ![osu! icon](/wiki/shared/mode/osu.png) [osu!](/wiki/Ranking_Criteria/osu!)
+  - ![osu!taiko icon](/wiki/shared/mode/taiko.png) [osu!taiko](/wiki/Ranking_Criteria/osu!taiko)
+  - ![osu!catch icon](/wiki/shared/mode/catch.png) [osu!catch](/wiki/Ranking_Criteria/osu!catch)
+  - ![osu!mania icon](/wiki/shared/mode/mania.png) [osu!mania](/wiki/Ranking_Criteria/osu!mania)
+
+## Beatmapping
+
+*Trang chính: [Beatmapping](/wiki/Beatmapping)*
+
+Beatmapping là quá trình của một người sáng tạo tạo ra beatmap. Quá trình này bao gồm: chọn nhạc, căn thời gian và tạo ra beatmap, và cuối cùng là thử beatmap. Nó cũng có thể bao gồm: thêm một video, thêm một [storyboard](/wiki/Storyboarding), và/hoặc thêm một [skin tự chọn](/wiki/Skinning). Tất cả những điều trên có thể được làm bởi cá nhân người sáng tạo hoặc với những người dùng khác, nhiều khi được gọi là *storyboarder*, *skinners*, và/hoặc *người sáng tạo được mời* dựa trên những điều họ làm. 
+
+## Modding
+
+*Trang chính: [Modding](/wiki/Modding)*
+
+Moddung là quá trình của những người chơi khác (còn được gọi là "modders") đánh giá beatmap của một người sáng tạo khác trong quá trình trờ đợi xếp hạng (hoặc chế độ WIP/cần giúp đỡ). Modding có một vai trò rất quan trọng trong quá trình đảm bảo chất lượng, bởi vì modding giúp những người sáng tạo sửa chữa những vấn đề với beatmap của họ.
```

Comment:
```suggestion
Modding là quá trình của những người chơi (còn được gọi là "người mod") đánh giá beatmap của một người sáng tạo khác trong giai đoạn chờ đợi xếp hạng (hoặc đang tiến hành/cần giúp đỡ). Modding có một vai trò rất quan trọng trong quá trình đảm bảo chất lượng, bởi vì modding giúp những người sáng tạo sửa chữa những vấn đề với beatmap của họ.
```
You should either use `người chơi khác... người sáng tạo` or `người chơi... người sáng tạo khác`.
For `modders`, you may keep as-is.